src/governance-object.h

```diff
@@ -48,6 +49,9 @@ static const int SEEN_OBJECT_ERROR_IMMATURE = 2;
 static const int SEEN_OBJECT_EXECUTED = 3; //used for triggers
 static const int SEEN_OBJECT_UNKNOWN = 4; // the default
 
+static const int GOVERNANCE_FILTER_MAX_ELEMENTS = 20000;
```

Comment:
redundant? should call on consensus.nGovernanceFilterElements when this is needed, right?